legal-api/src/legal_api/services/filings/validations/alteration.py

```diff
@@ -83,7 +83,8 @@ def company_name_validation(filing):
         validation_result = namex.validate_nr(nr_response)
 
         if not nr_response['requestTypeCd'] in ('CCR', 'CCP', 'BEC', 'BECV'):
-            msg.append({'error': babel('Alteration only available for Change of Name Name requests.'), 'path': nr_path})
+            msg.append({'error': babel('The name type associated with the name request number entered  \
+                                       cannot be used for this transaction type.'), 'path': nr_path})
```

Comment:
I'm not sure this gives you the expected output.

Reply:
Thanks Argus, yeah that makes sense, Thor also mentioned that i have blank spaces.... i have fixed it and will push my changes back up